WarehouseManagement.Core/Services/EntryService.cs

```diff
@@ -18,18 +18,24 @@ public EntryService(IRepository repository)
         this.repository = repository;
     }
 
-    public async Task CreateAsync(EntryFormDto model, string userId)
+    public async Task CreateAsync(ICollection<EntryFormDto> model, string userId)
```

Comment:
i just thought about it, maybe we can have validation for an entry that if you have more than 1 type inside your model, it should throw an error.
For example body comes with data looking like this:
```json
{
    "pallets": 5,
    "packages": 10,
    "pieces": 0
}
```

The reason is because an entry should contain only 1 type of quantity, it cant have more than one.

Reply:
fixed